src/storage.coffee

```diff
@@ -1,14 +1,16 @@
-import Basil from 'basil.js'
-
-store = new Basil(namespace: null)
-
-# a thin wrapper around basil.js for easy swapping
+# a thin wrapper around localStorage checking if localStorage is available
 class Storage
   constructor: (@namespace='alephbet') ->
-    @storage = store.get(@namespace) || {}
+    try
+      check = 'localstorage_check'
+      localStorage.setItem(check, check);
+      localStorage.removeItem(check);
+      @storage = JSON.parse(localStorage[@namespace])
```

Comment:
This can raise an error when `localStorage[@namespace]` is undefined. Maybe better write this here

```suggestion
      @storage = JSON.parse(localStorage[@namespace] || "{}")
```

Reply:
👍